examples/arduino_keyboard.rs

```diff
@@ -40,7 +40,9 @@ use atmega_usbd::{SuspendNotifier, UsbBus};
 use avr_device::{asm::sleep, interrupt};
 use usb_device::{
     class_prelude::UsbBusAllocator,
+    descriptor::lang_id::LangID::EN,
```

Comment:
This should be just `descriptor::lang_id::LangID` else rust complains of LangID not being a module